.cirrus.yml

```diff
@@ -368,13 +366,18 @@ build_success_task:
 bindings_task:
     name: "Test Bindings"
     alias: bindings
-    # Don't create task for PRs using [CI:DOCS] or [CI:BUILD]
     # Docs: ./contrib/cirrus/CIModes.md
+    # only when: - it is not a PR (we never want to skip on nightly tests); or
+    #            - CI:ALL title is set; or
+    #            - danger files are changed; or
```

Comment:
This subset is common to almost all the new filters. The duplication makes it very easy for a maintainer to understand when looking at ONE task, isolated; but much harder when looking at more than one. Did you consider something like a big comment at top describing the common portion, then reducing the individual comments to
```
# only when: - bindings test code is changed; or
#            - source code is changed AND NOT only test files; or
#            -  (see "ALWAYS RUN" above for standard list)
```

Reply:
that makes sense, my goal is to document he purpose of each file in CIModes.md and then I think linking that makes the most sense